### PR TITLE
Prepare the worker for docker images

### PR DESCRIPTION
**Pull request by mholthausen**
This PR 

- is used to prepare the existing workers `downloadNewDataFromURL` and `gunzipFile` as Docker images.

**Note**: 
- the amqp connection is changed here from `localhost` to `rabbitmq` (name of RabbitMQ service via docker-compose file)
- see the README files of the workers to build the images

The GitHub action to build and publish the images will be part of another PR.

@JakobMiksch @weskamm please note, will merge soon if there are no objections

### CODE REVIEW

File: src/downloadNewDataFromURL/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM node:alpine
+
+RUN apk update && apk add bash
```
Comment (JakobMiksch) [inline src/downloadNewDataFromURL/Dockerfile:3]: is `bash` really needed?

File: src/gunzipFile/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM node:alpine
+
+RUN apk update && apk add bash
```
Comment (JakobMiksch) [inline src/gunzipFile/Dockerfile:3]: is `bash` really needed?

**Review by JakobMiksch — APPROVED** [review]
I have not tested it, but the code looks alright. Note my comments inline.

**Comment by mholthausen** [comment]
Thanks! No, bash isn't needed. It was a leftover from testing but is removed now.

**Comment by JakobMiksch** [comment]
@mholthausen Feel free to merge.

**Pull request merged**